src/main/java/macrobase/diagnostics/ScoreDumper.java

```diff
@@ -0,0 +1,41 @@
+package macrobase.diagnostics;
```

Comment:
This file caught my eye - is there a particular reason this couldn't be integrated with src/test/java/macrobase/diagnostic/tasks/ScoreDumpDiagnostic.java?